ASWF/meetings/tsc/2020-11-16.md

```diff
@@ -0,0 +1,100 @@
+<!-- SPDX-License-Identifier: CC-BY-4.0 -->
+<!-- Copyright Contributors to the OpenColorIO Project. -->
+
+November 16, 2020
+
+Host: Doug Walker
+
+Rotating Secretary: Carol Payne
+
+Attendees:
+  * [x] Mark Boorer (_TSC_) - Industrial Light & Magic
+  * [x] Mei Chu (_TSC_) - Sony Pictures Imageworks
+  * [ ] Sean Cooper (_TSC ACES TAC Rep_) - DNEG
+  * [ ] Michael Dolan (_TSC Chair_) - Epic Games
+  * [x] Patrick Hodoul (_TSC_) - Autodesk
+  * [ ] John Mertic - Academy Software Foundation / Linux Foundation
+  * [x] Carol Payne (_TSC_) - Netflix
+  * [x] Mark Titchener (_TSC_) - Foundry
+  * [x] Carl Rand (_TSC_) - Weta Digital
+  * [x] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [x] Kevin Wheatley (_TSC_) - Framestore
+  * [x] Bernard Lefebvre - Autodesk
+  * [x] J Schulte - ILM
+  * [x] Matthias Scharfenberg - ILM
+  * [x] Larry Gritz - SPI
+  * [x] Aymee Kennett (Yeh) - Netflix
+
+Apologies:
+  Michael Dolan
+
+# **OCIO TSC Meeting Notes**
+
+* Named Color Transforms - Continued discussion
+    - Doug: Patrick, Bernard and I have been discussing the feedback and how to respond. We have made it possible to turn off the NamedTransforms - similar to an environment variable. Config author/ dev would be able to control whether or not the feature is visible to the user. Bernard has pushed that commit to the PR so it should be available for viewing.
+    - Mark: on/off isn't a true solution - we don't want to fracture the community. The opposition was because we felt that NamedTransforms opposed the way we want OCIO to work.
+    - Doug: would be interested to hear alternate proposals.
+    - Mark: the solutions already exist - you just have to utilize your working space as your reference space. 
+    - Doug: Not everyone is going to be the working space of the config. 
```

Comment:
be the working space of the config --> be solely using the working space of the config